docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -803,19 +803,26 @@ in the following sections:
 
 === Spring Cloud LoadBalancer Caching
 
-Apart from the basic `ServiceInstanceListSupplier` implementation that retrieves instances
-via `DiscoveryClient` each time it has to choose an instance, we provide a https://github.com/ben-manes/caffeine[Caffeine-backed]
-implementation.
-To make use of it, you need to have `com.github.ben-manes.caffeine:caffeine` in the classpath.
+Apart from the basic `ServiceInstanceListSupplier` implementation that retrieves instances via `DiscoveryClient` each time it has to choose an instance, we provide two caching implementations:
+
+* https://github.com/ben-manes/caffeine[Caffeine-backed]
+* https://github.com/stoyanr[Evictor-backed]
+
+If you have `com.github.ben-manes.caffeine:caffeine` in the classpath, Caffeine-based implementation will be used.
 The default setup includes `expireAfterWrite` set to 30 seconds and records set to soft references.
+The default `initialCapacity` is `256`.
+
+If you do not have Caffeine in the classpath, but have `com.stoyanr:evictor` in the classpath, Evictor-based implementation will be used.
```

Comment:
I wouldn't say if Evictor is on the classpath because it will be

Reply:
yes, but if someone is using the dependency directly, they will not have it. Will try to write it more expressely.